src/client/pages/Varen/useVarenListPage.hook.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  varenMeerInformatieLink,
+  tableConfig,
+  exploitatieVergunningAanvragen,
+  ListPageParamKind,
+} from './Varen-thema-config';
+import { VarenVergunningFrontend } from '../../../server/services/varen/config-and-types';
+import { isError, isLoading } from '../../../universal/helpers/api';
+import { addLinkElementToProperty } from '../../components/Table/TableV2';
+import { useAppStateGetter } from '../../hooks/useAppState';
+
+export function useVarenListPage(kind: ListPageParamKind) {
```

Comment:
Hier kun je of `useVarenThemaData` gebruiken of deze file verwijderen.

Het is, lijkt mij, niet nodig om `isLoading, isError en varenVergunningen`etc. allemaal opniuew te definiëren of om nogmaals ergens `addLinkElementToProperty` te moeten onderhouden.